gio/src/list_model.rs

```diff
@@ -103,8 +103,8 @@ impl<'a, T: IsA<glib::Object>> Iterator for ListModelIter<'a, T> {
     }
 
     fn size_hint(&self) -> (usize, Option<usize>) {
-        let n = (self.reverse_pos - self.i) as usize;
-        (n as usize, Some(n))
+        let n: usize = (self.reverse_pos - self.i) as _;
+        (n as _, Some(n))
```

Comment:
```suggestion
        (n, Some(n))
```

You already make it an `usize` before. This warning was actually correct

Reply:
You didn't change anything here :)

Reply:
I did one `git checkout --` too much.